apps/chat-e2e/src/tests/chatSelectionFunctionality.test.ts

```diff
@@ -0,0 +1,251 @@
+import { Conversation } from '@/chat/types/chat';
+import dialTest from '@/src/core/dialFixtures';
+import {
+  ExpectedConstants,
+  ExpectedMessages,
+  MenuOptions,
+} from '@/src/testData';
+import { Colors } from '@/src/ui/domData';
+import { expect } from '@playwright/test';
+
+dialTest(
+  '[UI] Check highlight of chat1 when chat2 is opened.\n' +
+    'Rename of chat1 when chat2 is opened.\n' +
+    'Compare mode is opened if to click on Compare for not selected chat.\n' +
+    'Replay chat1 when chat2 is opened.\n' +
+    'Playback chat1 when chat2 is opened.\n' +
+    'Export of chat1 when chat2 is opened.\n' +
+    'Delete of chat1 when chat2 is opened.\n' +
+    'Move to a folder of chat1 when chat2 is opened.\n' +
+    'Duplicate of chat1 when chat2 is opened.\n' +
+    'Share of chat1 when chat2 is opened',
+  async ({
+    dialHomePage,
+    conversationData,
+    dataInjector,
+    conversations,
+    conversationAssertion,
+    setTestIds,
+    localStorageManager,
+    compareConversation,
+    confirmationDialog,
+    folderConversations,
+    chatBarFolderAssertion,
+    shareModal,
+  }) => {
+    setTestIds(
+      'EPMRTC-934',
+      'EPMRTC-935',
+      'EPMRTC-936',
+      'EPMRTC-937',
+      'EPMRTC-3058',
+      'EPMRTC-938',
+      'EPMRTC-939',
+      'EPMRTC-940',
+      'EPMRTC-3059',
+      'EPMRTC-3060',
+    );
+    let firstConversation: Conversation;
+    let secondConversation: Conversation;
+    let replayConversation: string;
+    let playbackConversation: string;
+    let clonedConversation: string;
+
+    await dialTest.step('Create chat1 and chat2', async () => {
+      firstConversation = conversationData.prepareDefaultConversation();
+      conversationData.resetData();
+      secondConversation = conversationData.prepareDefaultConversation();
+      await dataInjector.createConversations([
+        firstConversation,
+        secondConversation,
+      ]);
+      await localStorageManager.setSelectedConversation(secondConversation);
+    });
+
+    await dialTest.step(
+      'Click on chat2 -> its history is opened on the central part of the screen',
+      async () => {
+        await dialHomePage.openHomePage();
+        await dialHomePage.waitForPageLoaded();
+      },
+    );
+
+    await dialTest.step('Hover over chat1', async () => {
+      await conversations.getEntityByName(firstConversation.name).hover();
+      await conversationAssertion.assertEntityBackgroundColor(
+        { name: firstConversation.name },
+        Colors.backgroundAccentSecondary,
+      );
+      await conversationAssertion.assertEntityDotsMenuState(
+        { name: firstConversation.name },
+        'visible',
+      );
+    });
+
+    await dialTest.step(
+      'Click on the menu and hover over any item at the bottom (e.g. Delete)',
+      async () => {
+        await conversations.openEntityDropdownMenu(firstConversation.name);
+        await conversations
```

Comment:
you can use `conversationDropdownMenu` fixture to select menu option

Reply:
done